src/facewidgetscene.cpp

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
**Reporter**: sneha-nd
**Rule**: whitespace/braces
**Severity**: WARN
**File**: src/facewidgetscene.cpp L26

{ should almost always be at the end of the previous line

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-37759655>*